server/config/twitterPassportRoutes.js

```diff
@@ -0,0 +1,42 @@
+import passport from 'passport';
+import { Strategy as TwitterStrategy } from 'passport-twitter';
+import dotenv from 'dotenv';
+
+import twitterCallback from '../helpers/twitterCallback';
+
+import models from '../models';
+
+const { User } = models;
+
+dotenv.config();
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((id, done) => {
+  User.findByPk(id).then((user) => {
+    done(null, user);
+  });
+});
+
+const twitterOptions = {
+  consumerKey: process.env.TWITTER_APP_ID,
+  consumerSecret: process.env.TWITTER_APP_SECRET,
+  callbackURL: 'http://localhost:3000/api/v1/auth/twitter/callback',
```

Comment:
Do we wanna hard-code this here? What if we are on Production?

Reply:
No. Can we set the port as an environment variable?

Reply:
Can't the callbackURL be set as an env and then you call the needed one based on your environment?

Reply:
I see what you mean. Should I make this change?